src/components/Welcome.js

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import React, { useState, useContext } from 'react';
 import { Link } from 'react-router-dom';
+import { ListContext } from '../context/ListContext';
+import './Welcome.css';
 
 export default function Welcome() {
+  const [userInputToken, setUserInputToken] = useState('');
+  const [error, setError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(null);
```

Comment:
Great overall approach to error handling! 

I think you can use only the errorMessage, and there is no need for the error, setError. You can check if the errorMessage is different than null, or just the ternary operator. 

It also seems that there is no clearing off the error, so maybe on handleChange you can clear the errorMessage